root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,21 +15,25 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet getMovie (String id) {
+	public Movie getMovie (String id) {
 		ResultSet rs = null;
 		String query = "select * from movies where id=?";
+		Movie movie = null;
 		try {
 			PreparedStatement stmt = con.prepareStatement(query);
 			stmt.setString(1, id);
 			rs = stmt.executeQuery();
+			if (rs.next()) {
+				movie = new Movie(rs.getString("id"), rs.getString("type"), rs.getString("name"), rs.getString("releasedate"), rs.getInt("duration"), rs.getString("filmrating"));
+			}
 		} catch (SQLException e) { 
-			throw new RuntimeException("failed to query db", e); 
+			throw new RuntimeException("failed to query movies table", e); 
 		}
 
-		return rs;
+		return movie;
 	}
 
-	public ResultSet updateMovie (Movie movie) {
+	public void updateMovie (Movie movie, String id) {
 		ResultSet rs = null;
 		String query = "update movies set type = ?, name = ?, releasedate = ?, duration = ?, filmrating = ? where id = ?";
```

Comment:
it would also be good to have these query strings use the constants that are created. If there's an easy way to do it then we can update it.